Makefile

```diff
@@ -42,6 +42,8 @@ TEST_UPGRADE_IMG ?= ""
 
 IS_GARDENER ?= false
 
+VERSION=$(shell git describe --tags --abbrev=0)
```

Comment:
I'm not sure if it would be better to have the version "dev" for everything except manifests and images created by the release jobs. It might be easier to control and understand, e.g. in this branch the version would be 1.9.0.
Using "dev" might be better to avoid confusion with incorrect version. Also the best we could do is using the latest version, what again is incorrect.
The only scenario where the current line is valid, is when running the release workflow. But in this case it might be better to use the value from RELEASE_TAG from the [publish_assets-script](https://github.com/kyma-project/api-gateway/blob/main/scripts/publish_assets.sh) and use "dev" in all other cases.

What do you think?